src/main/java/org/folio/clients/StorageClient.java

```diff
@@ -47,10 +52,10 @@
   }
 
   @NotNull
-  private URI prepareFullUri(List<String> ids, OkapiConnectionParams params) {
-    String query = ids.stream().map(s -> String.format(QUERY_PATTERN, s)).collect(Collectors.joining(" or "));
+  private URI prepareFullUri(List<String> ids, OkapiConnectionParams params, String recordPattern, String queryPattern) {
```

Comment:
Let's add limit parameters to URI to prevent limit issues in future to be sure that we always will receive a certain amount of records

Reply:
Ok, i will rework it